src/applications/disability-benefits/all-claims/tests/containers/ConfirmationPage.unit.spec.jsx

```diff
@@ -33,6 +33,22 @@ describe('Disability Benefits 526EZ <ConfirmationPage>', () => {
     tree.getByText('Claim ID number');
   });
 
+  it('should not render success with BDD SHA alert when not submitting BDD claim', () => {
+    const { queryByText } = render(
+      <ConfirmationPage
+        submissionStatus={submissionStatuses.succeeded}
+        {...defaultProps}
+        isSubmittingBDD={false}
+      />,
+    );
+
+    expect(
+      queryByText(
+        'Submit your Separation Health Assessment - Part A Self-Assessment now if you haven’t already',
+      ),
+    ).to.not.exist;
```

Comment:
Just a nitpick, and correct me if I'm wrong. In a niche situation where there is a copy change for the text, a future author is more likely to forget or miss updating this test because it'll still be passing. Would it be more effective to test for the presence of the text element in this case?

I completely agree with checking for copy on a positive test; it's just cause this one is a negative test